### PR TITLE
evs: Enable camera

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/camerabe/camerabe_git.bbappend [hunk 1]
```diff
@@ -0,0 +1,10 @@
+################################################################################
+# Renesas R-Car
+################################################################################
+SRCREV_rcar = "${AUTOREV}"
+
+SRC_URI_append_rcar = " git://github.com/xen-troops/camera_be.git;protocol=https;branch=master"
+
+EXTRA_OECMAKE_append_rcar = " -DWITH_DOC=OFF"
+
+RDEPENDS_${PN} = "libxenbe libconfig"
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/camerabe/camerabe_git.bbappend]: v4l2 dependencies?

Comment (rshym) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/camerabe/camerabe_git.bbappend]: Linrary "libv4l2" was added to dependencies. Library name was taken from makefiles of camera_be.

**Comment by rshym** [comment]
Added modification of weston.ini:
v4l2-renderer changed from /dev/media1 to /dev/vsp2-renderer

**Comment by andr2000** [comment]
So, after this change camera becomes mandatory HW to start DomU?

**Comment by rshym** [comment]
After merge of https://github.com/xen-troops/camera_be/pull/2 DomU can starts with/without camera HW.

**Comment by rshym** [comment]
Separate adding of vsp2-renderer and camera_front.
Fix rule for copying of camera_front.rules.
Add vsp2.rules and camera_front.rules to SRC_URI_append.

**Pull request merged**